libc/src/__support/macros/config.h

```diff
@@ -28,6 +28,7 @@
 #define LIBC_HAS_FEATURE(f) 0
 #endif
 
+#ifndef __GNUC__
```

Comment:
I think clang also define this macro.  Probably `#ifdef __clang__` is safer?

Reply:
Done.